scripts/start.js

```diff
@@ -364,7 +369,8 @@ function getUserInfo () {
         resolve({
           userName: username.slice(0, username.indexOf('@')),
           ssoToken: body.access_token,
-          domain: domain
+          domain: domain,
+          userId: userId,
```

Comment:
It would be better to always show the `Options` menu item.
In case the `userId` is missing, appropriate message shall be rendered instead of the option to see/modify the ssh-key.

Such a message should state
- what is happening
- what oVirt version is required (means with https://gerrit.ovirt.org/#/c/79579/)
- ideally even what current oVirt version is the portal connected to (see the About dialog, already there)

Reasons for that:
- the user knows there's something (s)he might do
- the dialog will be definitely reused for other config options